packages/medusa/src/api/routes/admin/draft-orders/index.ts

```diff
@@ -91,16 +91,64 @@ export const defaultAdminDraftOrdersFields: (keyof DraftOrder)[] = [
   "no_notification_order",
 ]
 
+/**
+ * @schema AdminPostDraftOrdersDraftOrderRegisterPaymentRes
+ * type: object
+ * properties:
+ *   order:
+ *     $ref: "#/components/schemas/DraftOrder"
```

Comment:
I know that this was the schema type in the original OAS, but looking at the returned response in `admin/draft-orders/register-payment.ts` it should be `Order` instead. Right?

Reply:
True!! Fixed in f145b774ee63e13c76926caadf0d9c5ee5161bfa